src/platform/site-wide/user-nav/components/HelpMenu.jsx

```diff
@@ -3,14 +3,19 @@ import React from 'react';
 
 import DropDownPanel from '@department-of-veterans-affairs/formation/DropDownPanel';
 import IconHelp from '@department-of-veterans-affairs/formation/IconHelp';
+
 import isBrandConsolidationEnabled from '../../../brand-consolidation/feature-flag';
+import isVATeamSiteSubdomain from '../../../brand-consolidation/va-subdomain';
 
 import facilityLocatorManifest from '../../../../applications/facility-locator/manifest';
 
 const FACILITY_LOCATOR_URL = facilityLocatorManifest.rootUrl;
 
 class HelpMenu extends React.Component {
   render() {
+    const facilityLocatorUrl = isVATeamSiteSubdomain()
```

Comment:
@rianfowler maybe we can use a similar check for the search action `/search` post launch